server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -586,6 +593,19 @@ private LookupResult lookup(SortedKeyValueIterator<Key,Value> mmfi, List<Range>
           mmfi.next();
         }
 
+        if (yield.hasYielded()) {
+          yielded = true;
+          Key yieldPosition = yield.getPositionAndReset();
```

Comment:
Should we check that yieldPosition is following the last KVEntry in the results object?